### ISSUE TITLE
Library no longer compatible with ipython

### ISSUE THREAD

**Issue by drvinceknight** [Axelrod-Python/Axelrod#1217]
After #1212 I now get:

```
Axelrod(master): ipython
Traceback (most recent call last):
  File "/home/vince/anaconda3/bin/ipython", line 7, in <module>
    from IPython import start_ipython
  File "/home/vince/anaconda3/lib/python3.6/site-packages/IPython/__init__.py", line 55, in <module>
    from .terminal.embed import embed
  File "/home/vince/anaconda3/lib/python3.6/site-packages/IPython/terminal/embed.py", line 16, in <module>
    from IPython.terminal.interactiveshell import TerminalInteractiveShell
  File "/home/vince/anaconda3/lib/python3.6/site-packages/IPython/terminal/interactiveshell.py", line 22, in <module>
    from prompt_toolkit.shortcuts import create_prompt_application, create_eventloop, create_prompt_layout, create_output
ImportError: cannot import name 'create_prompt_application'
```

This is because:

```
jupyter-console 5.2.0 has requirement prompt_toolkit<2.0.0,>=1.0.0, but you'll have prompt-toolkit 2.0.6 which is incompatible.
ipython 6.4.0 has requirement prompt-toolkit<2.0.0,>=1.0.15, but you'll have prompt-toolkit 2.0.6 which is incompatible.
```

Should we revert #1212 and wait for ipython to be compatible with a `prompt-toolkit>=2.0.0` or is there a better solution?

**Comment by drvinceknight** [comment]
On the `revert-prompt-toolkit` branch I've pushed 35bac5a which reverts the `prompt_toolkit` part of #1212. With that and after `pip install -r requirements.txt`: `ipython` now works. 

If we want that as a fix I'll open a PR and then we should open #1183 again and perhaps change the title to be clear that it's worth waiting for ipython to update their `prompt_toolkit` dependency.

**Comment by marcharper** [comment]
^ sounds good to me.
